packages/xpath/test/native/comparison-operator2.spec.ts

```diff
@@ -161,12 +161,12 @@ describe('Comparison operator (2)', () => {
 	});
 
 	describe('<, <=, >, >=', () => {
-		interface TestCaseInput {
+		interface InequalityTestCaseInput {
```

Comment:
I'd prefer `RelationalExprTestCaseInput`, as that's the term for these expressions in the [spec](https://www.w3.org/TR/1999/REC-xpath-19991116/#NT-RelationalExpr). (For future reference, it has been really helpful throughout the `xpath` package to hew close to spec terminology wherever possible.)